### DISCUSSION TITLE
(solved) Assertion failed: strcmp(version, "1.85 WIP") == 0 && "Mismatched version string!"

### DISCUSSION THREAD

**Discussion by rosidae** [ocornut/imgui#4893] [New Users Build/Link/Run issues ONLY!]
I get the error `Assertion failed: strcmp(version, "1.85 WIP") == 0 && "Mismatched version string!"` when I build with Visual Studio C++

I have installed imgui by running "vcpkg install imgui:x86-windows"
I am using d3d9 with ImGui

Help is appreciated

**Comment by rosidae** [comment]
It turns out vs wasn't detecting the right ImGui, it was detecting a git clone instead of the vcpkg version therefore causing a mismatch.

**Comment by ocornut** [comment]
This is exactly what this assertion is for, detect when you are including a .h that doesn't match the compiled version.
